CAVEATS.md

```diff
@@ -94,7 +138,19 @@ In PySide, the constructor for `QtGui.QRegExpValidator()` can just take a `QRegE
 In PyQt4 you are required to pass some form of a parent argument, otherwise you get a TypeError:
 
 ```python
-QtGui.QRegExpValidator(regex, None)
+# PySide
+>>> from Qt import QtCore, QtGui
+>>> regex = QtCore.QRegExp("\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}")
+>>> assert QtGui.QRegExpValidator(regex, None)
+>>> assert QtGui.QRegExpValidator(regex)
+```
+
+```python
+# PyQt4
+>>> from Qt import QtCore, QtGui
+>>> regex = QtCore.QRegExp("\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}")
+>>> assert QtGui.QRegExpValidator(regex, None)
+>>> assert_raises(TypeError, QtGui.QRegExpValidator, regex)  # Seems this works fine in PyQt4?
```

Comment:
> Which version of PyQt4 are you testing?

Hm. Actually, when glancing through the Travis build, there's no trace of an exact PySide or PyQt4 version anywhere.

@mottosso we should probably write that out somewhere (the exact version of the bindings used in the testing), perhaps via the new `build_caveats_tests.py`?